redbot/cogs/mod/settings.py

```diff
@@ -283,33 +296,97 @@ async def mentionspam_ban(self, ctx: commands.Context, max_mentions: int):
 
     @modset.command()
     @commands.guild_only()
-    async def deleterepeats(self, ctx: commands.Context, repeats: int = None):
+    async def deleterepeats(self, ctx: commands.Context, repeats: int = None, *channels):
```

Comment:
```suggestion
    async def deleterepeats(self, ctx: commands.Context, repeats: int = None, *channels: discord.TextChannel):
```
As I suggested in my previous review, it is **far** easier to just type hint the `channels` argument so dpy automatically converts it than to attempt to process it after the fact.